index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+
+    <title>Rolling Stone</title>
+</head>
+
+<body class = outer.container>
+    <nav class="navbar">
+        <ul>
+        <li>Case Studies</li>
+        <li>Services</li>
+        <li>Contact us</li>
+        </ul>
+    </nav>
+
+<header>
+    <section id="header-section" class="header-container">
+        <h1>Your website is an investment in the success of your business.</h1>
+        <p> We are here to help you succeed.</p>
+        <button>Let's work together</button>
+    </section>
+</header>
+
+<main id="outer-container" class="inner-container">
+
+    <section class="Rekupe">
+        <article>
+            <p> Rekupe is a digital agency in Los Angeles. </p>
+            <p> We use strategic UX, SEO optimization and beautiful visual designs to help your business grow. </p>
+            <p> We see every project as a fresh opportunity, pushing ourselves to create new digital experiences. </p>
+            <p class="para3"> Have a question? Click here to contact us! </p>
+        </article>
+    </section>
+
+   <section class="sectionrollingstone">
+       <article>
+            <div class="blokjeeen">
+                <div class="rsborder">
+                <h2>Rolling Stone</h2>
+                    <img class src="assets/squiggle_line.svg" alt="krulletjes onder tekst">
+                <h3> You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.
+                </h3>
+                </div>
+                    <span class="image-wrapper">
+                <img src= "/assets/work-image-01.jpeg" alt="foto van laptop en plant">
```

Comment:
De afbeeldingen laden bijna allemaal niet bij mij. Dat komt omdat het pad óf
```suggestion
                <img src= "assets/work-image-01.jpeg" alt="foto van laptop en plant">
```
of zo
```suggestion
                <img src= "./assets/work-image-01.jpeg" alt="foto van laptop en plant">
```
moet zijn 😛